docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+ase
+matplotlib
+furo
+rascaline @ https://github.com/luthaf/rascaline/archive/master.zip
```

Comment:
Are you sure you want to pull in the master? I would put a git hash in there until we have a release of rascaline =)

Reply:
Fine for me.